runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobServerDriver.java

```diff
@@ -90,9 +89,9 @@ String getFlinkMasterUrl() {
       name = "--sdk-worker-parallelism",
       usage = "Default parallelism for SDK worker processes (see portable pipeline options)"
     )
-    String sdkWorkerParallelism = PortablePipelineOptions.SDK_WORKER_PARALLELISM_PIPELINE;
+    Long sdkWorkerParallelism = 1L;
 
-    String getSdkWorkerParallelism() {
+    Long getSdkWorkerParallelism() {
```

Comment:
I would prefer `-1` as the "off value", trying to avoid null values (and a boxed long).

Reply:
Looking at `FlinkPipelineOptions`, we use the wrapper types everywhere. I would like to to keep it `Long` for consistency (does `long` even work)? Also, I prefer `null` to indicate "not present" or "not set" over -1.

Reply:
Indeed but I don't see why primitive types should not work. I'm ok with keeping it. If it can be null, it should at least be annotated as `Nullable`. IMHO Nulls in public interfaces should be avoided.

Reply:
👍  to using -1

Reply:
Actually, I think I misunderstood your comment about -1. Currently we use null to indicate that this config was not set in a particular place. There are two place this config can be set (as a job server option or as a pipeline option in python) and we need to be able to merge them, and set a default (1) if neither sets it. It appears that existing code uses boxed primitives and null for that and I've followed that precedent. I think it'd be worth re-examining this configuration code, but probably that's out of scope of this PR.

Reply:
In Dataflow we have been using null for unset object types and typically 0 for primitive numbers.